custom/templates/DefaultRevamp/email/api_register.html

```diff
@@ -1,16 +1,554 @@
+
+            </div>
+          <!--[if (mso)|(IE)]></td></tr></table><![endif]-->
+          </div>
+        </div>
+
+      </div>
+      <div>
+      <div class="layout one-col fixed-width stack" style="Margin: 0 auto;max-width: 600px;min-width: 320px; width: 320px;width: calc(28000% - 167400px);overflow-wrap: break-word;word-wrap: break-word;word-break: break-word;">
+        <div class="layout__inner" style="border-collapse: collapse;display: table;width: 100%;background-color: #ffffff;">
+        <!--[if (mso)|(IE)]><table align="center" cellpadding="0" cellspacing="0" role="presentation"><tr class="layout-fixed-width" style="background-color: #ffffff;"><td style="width: 600px" class="w560"><![endif]-->
+          <div class="column" style="text-align: left;color: #565656;font-size: 14px;line-height: 21px;font-family: Georgia,serif;">
+
+            <div style="Margin-left: 20px;Margin-right: 20px;Margin-top: 24px;">
+        <div style="font-size: 12px;font-style: normal;font-weight: normal;line-height: 19px;Margin-bottom: 20px;" align="center">
+          <img style="border: 0;display: block;height: auto;width: 100%;max-width: 900px;" alt="" width="560" src="https://namelessmc.com/uploads/template_banners/homepage_bg_trimmed.jpg" />
+        </div>
+      </div>
+
+            <div style="Margin-left: 20px;Margin-right: 20px;">
+      <div style="mso-line-height-rule: exactly;mso-text-raise: 11px;vertical-align: middle;">
+        <p class="size-20" style="Margin-top: 0;Margin-bottom: 20px;font-size: 17px;line-height: 26px;text-align: center;" lang="x-size-20"><strong>Welcome to [Sitename]</strong></p>
+      </div>
+    </div>
+
+            <div style="Margin-left: 20px;Margin-right: 20px;">
+      <div style="mso-line-height-rule: exactly;mso-text-raise: 11px;vertical-align: middle;">
+        <p class="size-16" style="Margin-top: 0;Margin-bottom: 0;font-size: 16px;line-height: 24px;" lang="x-size-16">Thanks for registering for&nbsp;<strong>[Sitename]</strong></p><p class="size-16" style="Margin-top: 20px;Margin-bottom: 20px;font-size: 16px;line-height: 24px;" lang="x-size-16">In order to get started please verify your account, after your account has been verified you can login on our website.</p>
+      </div>
+    </div>
+
+            <div style="Margin-left: 20px;Margin-right: 20px;">
+      <div class="btn btn--flat btn--medium" style="Margin-bottom: 20px;text-align: center;">
+        <![if !mso]><a style="border-radius: 4px;display: inline-block;font-size: 12px;font-weight: bold;line-height: 22px;padding: 10px 20px;text-align: center;text-decoration: none !important;transition: opacity 0.1s ease-in;color: #ffffff !important;background-color: #51b046;font-family: Georgia, serif;" href="[link]">Verify Account</a><![endif]>
```

Comment:
`[Link]` placeholder here instead of "Verify Account" is how it currently works, I think so that users will know the link theyre clicking is legitimate

Reply:
![image](https://user-images.githubusercontent.com/58806240/100263825-49614c80-2f4e-11eb-85ca-74bd90d17b32.png)